### ISSUE TITLE
Update Catalog Item Name and Description

### ISSUE THREAD

**Issue by beanh66** [ansibleplaybookbundle/import-vm-apb#5]
Need to finalize the catalog name and description. @aglitke Can you add a more detailed description? Do we need separate descriptions for the upstream and downstream versions? Are we okay with "Import Virtual Machine" as the catalog item name? 

@serenamarie125 FYI

**Comment by aglitke** [comment]
I am not sure what else to add for the top-level description.  What more needs to be said?

**Comment by xsgordon** [comment]
I believe that "Import Virtual Machine" is OK now that this encompasses both the from VMware and from URL options.

**Comment by aglitke** [comment]
@beanh66 Any recommendation for the added description text?

**Comment by beanh66** [comment]
@aglitke I'll check with @lizsurette and see if we can provide some suggestions.

**Comment by lizsurette** [comment]
@aglitke @beanh66 Yeah I think it looks pretty good considering there are really only two options for import at the moment: Import from image and Import from VMware vCenter VM. Each allow for importing as a template or importing as a new VM. Maybe something like this would be a full description?

"Import an existing image or VMware vCenter virtual machine as a new template or virtual machine."

Do we want to remove the "VIRTUALMACHINE" tag under the title? This seems to be used in other places to note the author of the catalog item. I can't think of anything better to put here but I do notice some items don't have it at all.

Also, it looks like other APBs link off to some additional documentation. Is there anything that we could point a user to if they'd like to read more?
